test/extensions/filters/http/cache/simple_http_cache/simple_http_cache_test.cc

```diff
@@ -250,10 +268,34 @@ TEST_F(SimpleHttpCacheTest, VaryResponses) {
   // Add second version and make sure we receive the correct one..
   const std::string Body2("accept is text/html");
   insert(move(second_value_vary), response_headers, Body2);
-  EXPECT_TRUE(expectLookupSuccessWithBody(lookup(RequestPath).get(), Body2));
+  EXPECT_TRUE(expectLookupSuccessWithBodyAndTrailers(lookup(RequestPath).get(), Body2));
 
   // Looks up first version again to be sure it wasn't replaced with the second one.
-  EXPECT_TRUE(expectLookupSuccessWithBody(first_value_vary.get(), Body1));
+  EXPECT_TRUE(expectLookupSuccessWithBodyAndTrailers(first_value_vary.get(), Body1));
+}
+
+TEST_F(SimpleHttpCacheTest, PutGetWithTrailers) {
+  const std::string request_path1("Name");
```

Comment:
This string should be something like `"/name"` so it provides a valid path (including the leading slash).